WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -448,6 +456,24 @@ private extension ManualTrackingViewController {
     }
 }
 
+// MARK: - UISheetPresentationControllerDelegate comformance
+//
+/// Asks permission to dismiss. We call this delegate method whenever the user attempts to dismiss the View Controller via the pull-down gesture.
```

Comment:
Super not a blocker, I don't think we need to add documentation because it is documented by apple on their sdk.